DESCRIPTION

```diff
@@ -13,48 +13,48 @@ Authors@R: c(person("Gao Wang",role = c("cre","aut"),
                   email = "dnachun@stanford.edu"))
 License: MIT + file LICENSE
 Imports:
-    GBJ,
     IRanges,
-    R.utils,
     R6,
     Rcpp,
-    Rfast,
     S4Vectors,
     bigsnpr,
     bigstatsr,
     coloc,
     data.table,
     doFuture,
     dplyr,
-    fsusieR,
     furrr,
     future,
-    glmnet,
-    harmonicmeanp,
     magrittr,
-    mashr,
     matrixStats,
-    mr.ash.alpha,
-    mr.mash.alpha,
-    mvsusieR,
-    pgenlibr,
     purrr,
-    qgg,
-    quantreg,
-    qvalue,
     readr,
     rlang,
-    snpStats,
```

Comment:
if we have bigstatsr not sure where we need snpStats --- did we use snpStats in the genotype data loader from PLINK format? if so, we still need that. That is currently the only tested way to load genotype from

Reply:
Yes, it's for loading PLINK formats, and VCFs which I will add support for later.